src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/service/IntersectionComputer.java

```diff
@@ -201,6 +206,19 @@ public static List<PathSegment> calculatePathIntersectionFromDest(Collection<Flo
         return Lists.reverse(getLongestIntersectionOfSegments(pathSegmentIterators));
     }
 
+    private OverlappingSegmentsStats computeIntersectionCounters(Set<Edge> edges, Set<SwitchId> switches) {
+        int edgesOverlap = Sets.intersection(edges, targetPathEdges).size();
+        int switchesOverlap = Sets.intersection(switches, targetPathSwitches).size();
+        return new OverlappingSegmentsStats(edgesOverlap,
+                switchesOverlap,
+                percent(edgesOverlap, targetPathEdges.size()),
+                percent(switchesOverlap, targetPathSwitches.size()));
+    }
+
+    private int percent(int n, int from) {
```

Comment:
`n` and `from` are no so informative. Why from? meaning total?

Reply:
It was there, I didn't rename it, but yeah you are right, I will rename it.

Reply:
Fixed.